components/tenant-fetcher/internal/model/tenant.go

```diff
@@ -4,7 +4,11 @@ import "github.com/kyma-incubator/compass/components/director/pkg/tenant"
 
 type TenantModel struct {
 	ID             string
+	Name           string
 	TenantId       string
-	TenantProvider string
-	Status         tenant.TenantStatus
+	ParentInternal string
```

Comment:
Probably ParentID is more meaningful name?

Reply:
Yes, it's more meaningful to have the ID suffix. 👍🏼 But is it necessary to remove the `Internal` part since it's a bit more descriptive this way?